docs/insights/coverage.md

```diff
@@ -0,0 +1,67 @@
+---
+id: coverage
+title: Measuring Your Test Coverage
+sidebar_label: Coverage
+description: See a breakdown of the browser and device variations covered by your tests.
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+The Coverage report allows Org Admins to view and download the breakdown of browsers and mobile devices the tests across their organization have run against. Having insight into your coverage data helps companies:
+
+* See at a glance whether their testing is in sync with the environments in which it is most often accessed
+* Design their test strategy to keep up with newly released browser and device versions
+* Utilize their concurrency to spread testing across test environments for greater efficiency and comprehensiveness
```

Comment:
Add bullet (pulled from line 32):

* Evaluate how evenly tests are being distributing across browsers and devices